<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMAER Configuration</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>551</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_key">
      <property name="text">
       <string>API Key</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_key"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_email">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_email">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_get_key">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>GetKey</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
